framework/include/problems/EigenProblem.h

```diff
@@ -143,6 +143,10 @@ class EigenProblem : public FEProblemBase
 
   virtual void initPetscOutput() override;
 
+  bool outputInverseEigenvalue() { return _output_inverse_eigenvalue; }
+
+  void outputInverseEigenvalue(bool inverse) { _output_inverse_eigenvalue = inverse; }
```

Comment:
did we decide to have a static function that GriffinApp can call to change the behavior of printing eigenvalues at the end?

Reply:
I knew that. But I realized it was not necessary to be static. This function does not need to be set earlier. 

But I could do that

Reply:
Then how Griffin change the default behavior? We do not want to ask users to set that parameter in the input all the time though.

Reply:
Suppress parameter or call this method in SNProblem.